### DISCUSSION TITLE
Erigon with Polygon, docker-compose

### DISCUSSION THREAD

**Discussion by miohtama** [erigontech/erigon#6771] [Q&A]
Hi all,

Are there any ready docker-composes to run Polygon node with Erigon? I found this [example from Chase](https://chasewright.com/erigon-polygon-archive-node/) but it seems to be quite manual approach? Because Polygon is split to different clients, the approach might need some careful instructions. 

We would like to switch our standard Polygon setup to Erigon-based one, as we have found out Erigon seems to be more robust solution over GoEthereum based solutions.

[Also we have been Erigon docker-compose example and it has been great](https://chasewright.com/erigon-polygon-archive-node/).

**Comment by AskAlexSharov** [comment]
there is nothing special. just add `--bor.heimdall="https://heimdall-api.polygon.technology/"` or `--bor.heimdall="https://heimdall-api-amoy.polygon.technology/"`
